### PR TITLE
Standard list item

### PR DESCRIPTION
**Pull request by k9101**
Added StandardListItem component

Examples under misc.

### CODE REVIEW

File: components/StandardListItem/StandardListItemStyles.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.StandardListItem {
+
+  display: -webkit-flex;
```
Comment (vikasrohit) [inline components/StandardListItem/StandardListItemStyles.scss]: Do we really need webkit specific css? We are good if it works, without specific css, with last 2 versions of the webkit browsers. If you agree, please remove webkit specific rules from other places as well.

Comment (k9101) [inline components/StandardListItem/StandardListItemStyles.scss]: Good point, I forgot that we're only supporting current browser versions

File: components/StandardListItem/StandardListItemStyles.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  display:         flex;
+  -webkit-align-items: center;
+          align-items: center;
+
+  text-align: center;
+
+  width: 100%;
+
+  // background-color: honeydew;
+  padding: 20px 15px;
```
Comment (vikasrohit) [inline components/StandardListItem/StandardListItemStyles.scss]: Can you please reduce it to have 15px from top/bottom and 20px from left/right? And further, this should change for the mobile view (<768px) to 12px from top/bottom and 15px from left/right.

File: components/StandardListItem/StandardListItemStyles.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.StandardListItem {
+
+  display: flex;
+  align-items: center;
+
+  text-align: center;
+
+  width: 100%;
+
+  padding: 20px 15px;
```
Comment (vikasrohit) [inline components/StandardListItem/StandardListItemStyles.scss]: Can you please change padding to 15px from top/bottom and 20px from left/right? Further, for mobile (<768px) view, please change it to 12px from top/bottom and 15px from left/right.

**Comment by vikasrohit** [comment]
Please make sure you are using 2 spaced indention. I am seeing some non 2 spaced indents in scss file.
Overall looks good. Made some comments, please review them.

**Comment by k9101** [comment]
I think I fixed the indentation (on .top .icon width & height), were there other spots that you were referring to?

**Comment by vikasrohit** [comment]
Now it seems fine. :+1:  I think, I commented, while you were fixing. :)

**Comment by vikasrohit** [comment]
@k9101 merging the PR, however, please keep your branch open until the challenge closes. We may require minor adjustments to the component as it is consumed by other components during the challenge duration.

Btw, @k9101, how much comfortable you are with unit testing these components?

**Comment by vikasrohit** [comment]
@nlitwin @samsep @aselbie @vietnogi Please feel free to suggest anything that can be improved here. We can have those changes incorporated in final fixes.

**Comment by k9101** [comment]
@vikasrohit Ok great, I'll keep mine open just in case.

I've haven't done unit test in React (I have with other platforms) before, but it looks pretty straight forward with either Jest or TestUtils.

**Comment by vikasrohit** [comment]
Awesome. We would come back for unit tests.

**Pull request merged**